src/services/connectorService.ts

```diff
@@ -43,7 +43,13 @@ export class ConnectorService {
     }
 
     handleAuthProcess(ingestIp: string, obsName: string, groupCode: string, leftTeam: AuthTeam, rightTeam: AuthTeam, key: string, win: Electron.Main.BrowserWindow) {
-        this.INGEST_SERVER_URL = `https://${ingestIp}:5100`;
+        if (RegExp('(http|https)://[^/]+:[0-9]+').test(ingestIp)) {
+            this.INGEST_SERVER_URL = `${ingestIp}`
+        } else if (RegExp('.+:.+').test(ingestIp) && !(ingestIp.startsWith('http'))) {
```

Comment:
If we already use a .startsWith to check for http instead of doing it all with one regex, we can also just use a .includes to check for the : instead of a regex.